roles/openshift_examples/defaults/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+# By default install rhel and xpaas streams on enterprise installs
+openshift_examples_load_centos: "{{ openshift_deployment_type != 'enterprise' }}"
+openshift_examples_load_rhel: "{{ openshift_deployment_type == 'enterprise' }}"
+openshift_examples_load_db_templates: true
+openshift_examples_load_xpaas: false
+openshift_examples_load_quickstarts: true
```

Comment:
@brenton These are the variables to set on a master should anyone wish to override the defaults. We'll update the xpaas defaults in a few days once they've signed off.